### DISCUSSION TITLE
Anyone else seeing this? (Firefox: Webpage slowing down browser)

### DISCUSSION THREAD

**Discussion by brisingraerowing** [community/community#37428] [Other Feature Feedback, Questions, & Ideas]
I was browsing various repos, and Firefox would briefly hang, then display a message about the page slowing down the browser and an option to kill the offending script. Doing so puts the following message in the DevTools console:

```text
Script terminated by timeout at:
66547/subscribe/o/c<@https://github.githubassets.com/assets/behaviors-6e50c7354882.js:1:18232
66547/subscribe/o/o<@https://github.githubassets.com/assets/behaviors-6e50c7354882.js:1:18337
o@https://github.githubassets.com/assets/behaviors-6e50c7354882.js:1:18447
```

I'm not sure if this is on my end (borked extension?) or something has gone wrong with GH itself.

**Comment by radar-blip** [comment]
I'm using Firefox. Not seeing this issue here.

**Comment by tropicbliss** [comment]
I am currently experiencing the same issue. Moved to edge and it hangs the whole browser too. There must be an infinite loop somewhere.
